scripts/parse_live.py

```diff
@@ -9,7 +9,7 @@
 `snippets` table.
 
 Usage:
-    parse_live.py <pageid-file> [--timeout=<n>]
+    parse_live.py [<pageid-file>] [--timeout=<n>]
```

Comment:
We typically want to process both a page ids file and CD, right? Shouldn't we just leave this as a mandatory argument then?

Reply:
So you mean we want to process both inputs from page ids file and CD at the same time? 
The reason I change the argument is that I assume we’ll call the script twice, one for processing page ids file and one for processing CD, and we distinguish them by whether we pass <pageid-file> argument.
But I feel it would be better if we process them at the same time.

Reply:
I'm also fine with having separate invocations, but then we'll want to update `scripts/update_db_tools_labs.py` to actually call the script twice. If you go that way, please include that change in this PR.

I'd suggest making things explicit like turning this line into:

```
parse_live.py (--import_from_citationdetective | <pageid-file>) [--timeout=<n>]
```

See the documentation for [doctopt](docopt.org), the cool little library that is processing this string to handle our command line arguments.